app/client/src/WidgetQueryGenerators/MySQL/index.ts

```
@@ -196,15 +196,15 @@ export default abstract class MySQL extends BaseQueryGenerator {
 
-  public static build(
+  public static async build(
     widgetConfig: WidgetQueryGenerationConfig,
     formConfig: WidgetQueryGenerationFormConfig,
     pluginInitalValues: { actionConfiguration: ActionConfigurationSQL },
   ) {
     const allBuildConfigs = [];
 
     if (widgetConfig.select) {
-      allBuildConfigs.push(this.buildSelect(widgetConfig, formConfig));
+      allBuildConfigs.push(await this.buildSelect(widgetConfig, formConfig));
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)